vhdbuilder/packer/install-dependencies.sh

```diff
@@ -293,16 +293,10 @@ done
 
 if [[ $OS == $UBUNTU_OS_NAME ]]; then
 NVIDIA_DEVICE_PLUGIN_VERSIONS="
-1.11
-1.10
+v0.9.0
```

Comment:
super nit: it seems like the pattern for install-dependencies is to append the "v" explicitly when constructing the image/pkg url, not in the VERSIONS array